pyomo/common/collections/orderedset.py

```diff
@@ -9,42 +9,30 @@
 #  This software is distributed under the 3-clause BSD License.
 #  ___________________________________________________________________________
 
-from collections.abc import MutableSet
 from collections import OrderedDict
+from collections.abc import MutableSet
+from pyomo.common.autoslots import AutoSlots
 
 
-class OrderedSet(MutableSet):
+class OrderedSet(AutoSlots.Mixin, MutableSet):
     __slots__ = ('_dict',)
 
     def __init__(self, iterable=None):
         # TODO: Starting in Python 3.7, dict is ordered (and is faster
```

Comment:
I know this comment is old but it's (1) confusing - what is actually "TODO" about this? - and (2) we now only support 3.8+, so does some sort of action need to be taken?